handbook/engineering/2021_org.md

```diff
@@ -1,46 +1,32 @@
-# 2021 Q1 Organizational Plan
+# 2021 Engineering Organization Plan
 
 Defining a future vision for the organization is necessary to effectively grow the engineering team. It is understood that all plans change, and this document will be kept up to date with these changes. This is our written plan to align the team and clarify hiring and growth expectations.
 
+[2021 headcount plan](https://docs.google.com/spreadsheets/d/1UG4aLB5S8yOSOByia2EQgpZkoPxG5eVfqQqgs1f71Vw/edit#gid=2097845084)
+
 - VP Engineering
-  - Director of Engineering: Core Products
-    - [Search Team](search/index.md)
-      - Engineering Manager: Search
-      - Product Manager: Search
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Campaigns Team](campaigns/index.md)
-      - Engineering Manager: Campaigns
-      - Product Manager: Campaigns
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Code Intelligence Team](code-intelligence/index.md)
-      - Engineering Manager: Code Intel
-      - Product Manager: Code Intel
-      - Backend Engineers: 3
-      - Frontend Engineers: 2
-    - [Extensibility Team](web/index.md)
-      - Engineering Manager: Extensions
-      - Product Manager: Extensions
-      - Frontend Engineers: 4
-    - [Code Host Integrations Team](web/index.md)
-      - Engineering Manager: Code Host Integrations
-      - Product Manager: Code Host Integrations
-      - Frontend Engineers: 4
-  - Director of Engineering: Core Services
-    - [Backend Infrastructure Team](backend-infrastructure/index.md)
-      - Engineering Manager: Backend Infrastructure
-      - Backend Engineers: 4
-      - Dev Ops Engineers: 2
-    - [Distribution Team](distribution/index.md)
-      - Engineering Manager: Distribution
-      - Product Manager: Distribution
-      - Dev Ops Engineers: 4
-      - Backend Engineers: 2
-    - [Security Team](security/index.md)
-      - Engineering Manager: Security
-      - Backend Engineers: 3
-      - Security Engineers: 3
-    - [Web Infrastructure Team](web/index.md)
-      - Engineering Manager: Web Infrastructure
-      - Frontend Engineers: 4
+  - Director of Engineering - Product
+    - [Search Core](search/index.md#search-core) (future split of current Search team)
+    - [Search Growth](search/index.md#search-growth) (future split of current Search team)
+    - [Campaigns](campaigns/index.md)
+    - [Code Intelligence](code-intelligence/index.md)
+    - [Code Insights](web/index.md#code-insights) (future team, currently owned by Web team)
+    - [Extensions and Integrations](web/index.md#extensions-and-integrations) (future team, currently owned by Web team)
+  - Director of Engineering - Infrastructure
+    - [Cloud](cloud/index.md)
+    - [Deployment and Operations Platform](distribution/index.md#deployment-and-operations-platform)
```

Comment:
Not sure if better, but the previous name is very long.

```suggestion
    - [Production Platform](distribution/index.md#deployment-and-operations-platform)
```

Reply:
I think Distribution does better capture what is described as the objective and scope of the team. I dont think its a deal-breaker and we can rename in the future.

Reply:
Thanks for the feedback, I changed this.